src/Pages/Subscribe/PaymentButton.js

```diff
@@ -0,0 +1,95 @@
+import { useHistory } from "react-router";
+import styled from "styled-components";
+import Swal from "sweetalert2";
+
+export default function PaymentButton(props) {
+  const history = useHistory();
+  const userCell = window.localStorage.getItem("userCell");
+  const {
+    isAllMandatoryChecked,
+    selectedPaymentPG,
+    productToOrder,
+    finalPrice,
+  } = props;
```

Comment:
비구조화 하리셜면 컴포넌트명에서 바로 하시는게 깔끔할 것 같습니다!
```suggestion
export default function PaymentButton({
    isAllMandatoryChecked,
    selectedPaymentPG,
    productToOrder,
    finalPrice}) {
  const history = useHistory();
  const userCell = window.localStorage.getItem("userCell");
```

Reply:
네~ 적용했습니다